myscripts.js

```diff
@@ -1,99 +0,0 @@
-function goFunction() {
-    //search();
-    var searchTerm = document.getElementById("searchText").value;
```

Comment:
Be careful when you access DOM using direct statement like this. You need to make sure DOM is ready when you execute this code. One way to do this by including the script at the end of the html file